### ISSUE TITLE
 TTGO LilyGO T-ETH-POE support

### ISSUE THREAD

**Issue by Mingyu-Kim** [platformio/platform-espressif32#475]
Can you add TTGO's T-ETH-POE support?

Board info
https://github.com/Xinyuan-LilyGO/LilyGO-T-ETH-POE
http://www.lilygo.cn/prod_view.aspx?TypeId=50033&Id=1307&FId=t3:50033:3

Example
https://github.com/Xinyuan-LilyGO/LilyGO-T-ETH-POE/tree/master/example/eth

Thank you

**Comment by cnoork** [comment]
I would also like that the board is added to PlatformIO

**Comment by Lgeorge-V** [comment]
I would greatly appreciate it if this board was added to PlatformIO.

**Comment by WombatHollow** [comment]
Yes please to adding ttgo boards including this one to platformio.   PLEASE

**Comment by PurP13cray0n** [comment]
Please add this board to the list of boards supported

**Comment by valeros** [comment]
There is no proper variant file for this board in the official Arduino core for ESP32 at the moment, so it needs to be added to this repo https://github.com/espressif/arduino-esp32 in the first place.

**Comment by PurP13cray0n** [comment]
Thank you for directing me towards the underlying problem, I have made a feature request in the Official Arduino core for ESP32.
[https://github.com/espressif/arduino-esp32/issues/9159](url)
